manageiq-gems-pending.gemspec

```diff
@@ -50,7 +50,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "net-sftp",                "~>2.1.2"
   s.add_runtime_dependency "nokogiri",                "~>1.7.1"
   s.add_runtime_dependency "openscap",                "~>0.4.3"
-  s.add_runtime_dependency "ovirt",                   "~>0.15.1"
+  s.add_runtime_dependency "ovirt",                   "~>0.16.0"
```

Comment:
This is becoming a problem with several providers.  I think we should loosen the dependency here and this should become:
```ruby
s.add_runtime_dependency "ovirt" # Version specified by manageiq-providers-ovirt
```
@Fryguy Any thoughts?